### DISCUSSION TITLE
Troubleshooting API calls in the dev environment

### DISCUSSION THREAD

**Discussion by kentcooper** [Fiserv/Support#530] [Q&A]
Is there any way to troubleshoot why an API call is failing? I've been attempting everything I can think of to make a successful call to https://cert.api.fiservapps.com/ch/payments-vas/v1/security/credentials but I constantly get the response below no matter what I try and I have no idea how to tell why the call is bad.

{
    "gatewayResponse": {
        "transactionProcessingDetails": {
            "apiTraceId": "0e43129eee6b4477a357724f0aaba0ee",
            "clientRequestId": "3bee3f58-3ee8-4e31-a4e3-f4612483d70b"
        }
    },
    "error": [
        {
            "type": "APIM",
            "message": "Bad request."
        }
    ]
}

**Comment by wesfincher** [comment]
Did you ever figure this out?
